Remove PostCSS warnings

Currently, we have below warnings from postCSS when running webpack. We can add either add a basic plugin like auto prefixer or just remove postCSS
```
[INFO] Using 1 worker with 2048MB memory limit
[ERROR] You did not set any plugins, parser, or stringifier. Right now PostCSS do nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
[ERROR] You did not set any plugins, parser, or stringifier. Right now PostCSS do nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
[ERROR] You did not set any plugins, parser, or stringifier. Right now PostCSS do nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
[ERROR] You did not set any plugins, parser, or stringifier. Right now PostCSS do nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
[ERROR] You did not set any plugins, parser, or stringifier. Right now PostCSS do nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
[ERROR] You did not set any plugins, parser, or stringifier. Right now PostCSS do nothing. Pick plugins for your case on https://www.postcss.parts/ and use them in postcss.config.js.
[INFO]  DONE  Compiled successfully in 12230ms23:04:29

```

--- a/generators/client/files-react.js
+++ b/generators/client/files-react.js
@@ -58,7 +58,7 @@ const files = {
     sass: [
         {
             condition: generator => generator.useSass,
-            templates: [{ file: 'postcss.config.js', method: 'copy' }]
+            templates: ['postcss.config.js']
         },
         {
             condition: generator => generator.useSass && generator.enableI18nRTL,

--- a/generators/client/templates/angular/package.json.ejs
+++ b/generators/client/templates/angular/package.json.ejs
@@ -74,6 +74,7 @@
     <%_ } _%>
     "angular-router-loader": "0.8.5",
     "angular2-template-loader": "0.6.2",
+    "autoprefixer": "9.2.0",
     "browser-sync": "2.24.6",
     "browser-sync-webpack-plugin": "2.2.2",
     "cache-loader": "1.2.2",

--- a/generators/client/templates/angular/postcss.config.js.ejs
+++ b/generators/client/templates/angular/postcss.config.js.ejs
@@ -17,5 +17,7 @@
  limitations under the License.
 -%>
 module.exports = {
-    plugins: []
+    plugins: [
+        require('autoprefixer')
+    ]
 }

--- a/generators/client/templates/react/package.json.ejs
+++ b/generators/client/templates/react/package.json.ejs
@@ -84,6 +84,7 @@ limitations under the License.
     "@types/selenium-webdriver": "3.0.10",
     <%_ } _%>
     "@types/webpack-env": "1.13.6",
+    "autoprefixer": "9.2.0",
     "browser-sync": "2.24.6",
     "browser-sync-webpack-plugin": "2.2.2",
     "cache-loader": "1.2.2",

--- a/generators/client/templates/react/postcss.config.js.ejs
+++ b/generators/client/templates/react/postcss.config.js.ejs
@@ -1,3 +1,23 @@
+<%#
+ Copyright 2013-2018 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
 module.exports = {
-  plugins: []
+  plugins: [
+    require('autoprefixer')
+  ]
 };

